tests/e2e/state.go

```diff
@@ -199,6 +199,16 @@ func (tr TestRun) getBlockHeight(chain chainID) uint {
 }
 
 func (tr TestRun) waitBlocks(chain chainID, blocks uint, timeout time.Duration) {
```

Comment:
May want to rename this to newBlocks or something since it doesn't always wait

Reply:
I think it's better like this - even with CometMock, the name still gets at what it does, namely after this is done executing a number of blocks will have passed